Rect face # origin is the top-left positions of face rectangle
Point2D face_center # center point of a face area
Rect[] eyes # origin is the top-left positions of each eyes
Point2D[] eyes_center # center points of each eyes
